app/views/clients/_form.html.slim

```diff
@@ -13,10 +13,7 @@
           = f.input :last_name
       .row
         .col-xs-12.col-md-6
-          = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
-                                 end_year: Date.today.year,
-                                 include_blank: true
-
+        = render 'date_of_birth_select', f: f
```

Comment:
Ok,  I'll have a look into that. But I't has no priority, because this PR is not bringing any functionality and doesn't fix anything. I'll put that on hold for now...